networks/movement/movement-client/src/bin/e2e/load_alice_bob.rs

```diff
@@ -86,107 +88,71 @@ static FAUCET_URL: Lazy<Url> = Lazy::new(|| {
 			self.id,
 			alice.address().to_hex_literal(),
 			bob.address().to_hex_literal()
 		);
 
-		tracing::info!("{} Before alice fund", self.id);
-		self.log_exec_info(&format!("{} Before alice fund", self.id));
 		// Create the accounts on chain, but only fund Alice.
-		faucet_client.fund(alice.address(), 100_000_000).await?;
-		tracing::info!("{} Before Bod create_account", self.id);
-		self.log_exec_info(&format!("{} Before Bod create_account", self.id));
+		faucet_client.fund(alice.address(), 100_000_000_000).await?;
 		faucet_client.create_account(bob.address()).await?;
-		tracing::info!("{} After Bod create_account", self.id);
-		self.log_exec_info(&format!("{} After Bod create_account", self.id));
-
-		// Print initial balances.
-		tracing::info!(
-			"Scenario:{}\n=== Initial Balances ===\nAlice: {:?}\nBob: {:?}",
-			self.id,
-			coin_client
-				.get_account_balance(&alice.address())
-				.await
-				.context("Failed to get Alice's account balance")?,
-			coin_client
-				.get_account_balance(&bob.address())
-				.await
-				.context("Failed to get Bob's account balance")?
-		);
 
 		// Have Alice send Bob some coins.
 		let txn_hash = coin_client
-			.transfer(&mut alice, bob.address(), 1_000, None)
+			.transfer(&mut alice, bob.address(), 1_000_000, None)
 			.await
 			.context("Failed to submit transaction to transfer coins")?;
 		rest_client
 			.wait_for_transaction(&txn_hash)
 			.await
 			.context("Failed when waiting for the transfer transaction")?;
+		tracing::info!("Scenario:{} prepare done. account created and founded", self.id,);
 
-		// Print intermediate balances.
-		tracing::info!(
-			"Scenario:{}\n=== Intermediate Balances ===\nAlice: {:?}\nBob: {:?}",
-			self.id,
-			coin_client
-				.get_account_balance(&alice.address())
-				.await
-				.context("Failed to get Alice's account balance the second time")?,
-			coin_client
-				.get_account_balance(&bob.address())
-				.await
-				.context("Failed to get Bob's account balance the second time")?
-		);
+		self.alice = Some(alice);
+		self.bob = Some(bob);
+		Ok(())
+	}
 
-		self.log_exec_info(&format!("Scenario:{} ended", self.id));
+	async fn run(&mut self) -> Result<()> {
+		let rest_client = Client::new(NODE_URL.clone());
+		let coin_client = CoinClient::new(&rest_client); // Print initial balances.
 
-		// Have Alice send Bob some coins.
-		let txn_hash = coin_client
-			.transfer(&mut alice, bob.address(), 1_000, None)
-			.await
-			.context("Failed to submit transaction to transfer coins")?;
-		rest_client
-			.wait_for_transaction(&txn_hash)
-			.await
-			.context("Failed when waiting for the transfer transaction")?;
+		let alice = self.alice.as_mut().unwrap();
+		let bob = self.bob.as_mut().unwrap();
 
-		// Print intermediate balances.
-		tracing::info!(
-			"Scenario:{}\n=== Intermediate Balances ===\n Alice: {:?}\n Bob: {:?}",
-			self.id,
-			coin_client
-				.get_account_balance(&alice.address())
+		for _ in 0..2 {
```

Comment:
Why is this done two times in one test run?

Reply:
It's to add more Tx in the scenario. Depending on the load we want to create, we can increase or decrease the number of loop. 2 is enough to get the issues we've in bardock. When they will be solved, we can try with more Tx.

Reply:
In this scenario, to scale the load we use Tx. Adding more scenario add more Tx, but each scenario needs to create and fund 2 accounts. So it takes time and can cause some issue with the faucet. By increasing the number of Tx in a scenario, we allow scaling without increasing the account creation.
I keep it because depending on the test we want to do we can need to tweak a little these parameters.
2 is a good compromise between not adding too much TX and using the number of scenario created as variable parameters and keeping the loop when someone need it.